sdk/communication/Azure.Communication.CallingServer/README.md

```diff
@@ -35,6 +35,14 @@ var connectionString = "<connection_string>"; // Find your Communication Service
 CallingServerClient callingServerClient = new CallingServerClient(connectionString);
 ```
 
+```C# Snippet:Azure_Communication_CallingServer_Tests_Samples_CreateCallingServerClientWithToken
+string endpoint = "<endpoint_url>";
+string userAssignedClientId = "<your managed identity client Id>";
+var tokenCredential = new DefaultAzureCredential(new DefaultAzureCredentialOptions { ManagedIdentityClientId = userAssignedClientId });
```

Comment:
Why is the `ManagedIdentityClientId` required here?

Reply:
It shouldn't be there. Removed it, thanks for catching.